cabot/metricsapp/models/base.py

```diff
@@ -123,3 +123,15 @@ def set_fields_from_grafana(self, fields):
             self.high_alert_value = high_alert_value
 
         self.save()
+
+    def get_status_image(self):
+        """Return a Grafana png image for the check if it exists"""
+        if self.grafana_panel is not None:
```

Comment:
Shouldn't we have used a derived class `GrafanaStatusCheckBase(MetricsStatusCheckBase)` and put all grafana related checks inside it?

It gives us perfect abstraction.

Also, when we do cloudwatch integration, this will become very important...